From c282c858166ab32ae4e2ad296d62f7a70001c322 Mon Sep 17 00:00:00 2001
From: Grace Cham <hscham@chromium.org>
Date: Thu, 19 May 2022 12:46:04 +0900
Subject: [PATCH] Add base::Value::TakeList

It was removed in r967544 (crrev.com/c/3440910) but the new
base::Value::List type will not be added until r971315
(crrev.com/c/3443197).
Allow use of this deprecated function so that we can migrate to the new
types directly after uprev.

Change-Id: I1b688c9c6b58894b7aaa51375b516fd659489556
---
 base/values.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/base/values.h b/base/values.h
index f1b25181d2..c8e361628c 100644
--- a/base/values.h
+++ b/base/values.h
@@ -673,6 +673,7 @@ class BASE_EXPORT GSL_OWNER Value {
   // DEPRECATED: prefer direct use of `base::Value::List` where possible, or
   // `std::move(value.GetList())` otherwise.
   DeprecatedListStorage TakeListDeprecated() &&;
+  ListStorage TakeList() && { return std::move(*this).TakeListDeprecated(); }
 
   // Appends `value` to the end of the list.
   //
-- 
2.36.1.124.g0e6072fb45-goog

